pkg/commands/version.go

```diff
@@ -0,0 +1,13 @@
+package commands
+
+import (
+	"fmt"
+	"runtime"
+)
+
+func VersionCommand(commands *[]string, flags *map[string]interface{}) error {
```

Comment:
Each Command class could be implement a BaseCommand struct like this:

```go
type Command struct {
	Name        string
	Description string
    Run     func(commands *[]string, flags *map[string]interface{})
}
```

Reply:
Great idea! I'll implement that tonight thank you

Reply:
Updated now, got excited :D